index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="style.css" />
+  <title>Cluedo Reminisce</title>
+</head>
+
+<body>
+    <h1> CL<span class="red">U</span>EDO </h1>
+    <h3>The classic <span class="yellow">mystery</span> game</h3>
+    <section class="cards">
+      <article class="killer-deck" id="killerCard">
+        <p class="title-font">KILLER</p>
+        <img id="killerImage" src="" height="200" />
+        <p class="second-title-font" id="killerName"></p>
+        <p class="third-title-font" id="killerAge"></p>
+        <p class="third-title-font" id="killerOccupation"></p>
+        <p class="third-title-font" id="killerDescription"></p>
+        <p class="third-title-font" id="killerFavouriteWeapon"></p>
+      </article>
+      <article class="weapon-deck" id="weaponCard">
+        <p class="title-font">WEAPON</p>
+        <img class="weapon-image" id="weaponImage" src="" height="130" />
+        <p class="second-title-font" id="weaponName"></p>
+        <p class="third-title-font" id="weaponWeight"></p>
+      </article>
+      <article class="room-deck" id="roomCard">
+        <p class="title-font">ROOM</p>
+        <img class="room-image" id="roomImage" src="" height="130" />
+        <p class="second-title-font" id="roomName"></p>
+      </article>
+    </section>
+
+  <h2 class="mystery-reveal-text" id="mystery"></h2>
+
+  <button onclick="revealMystery()">REVEAL THE CRIME</button>
+
+  <script src="clue.js"></script>
```

Comment:
It's good that you have linked the script in the end of the HTML-file so the content can load before the script does :+1: